plenum/test/node_catchup_with_3pc/test_stashing_3pc_while_catchup_only_checkpoints.py

```diff
@@ -141,8 +141,8 @@ def test_3pc_while_catchup_with_chkpoints_only(tdir, tconf,
         )
 
         # check that catch-up was started twice
-        assert lagging_node.spylog.count(Node.allLedgersCaughtUp) == initial_all_ledgers_caught_up + 2
-        assert lagging_node.spylog.count(Node.start_catchup) == 2
+        assert lagging_node.spylog.count(Node.allLedgersCaughtUp) == initial_all_ledgers_caught_up + 1
```

Comment:
Why is it 1 again? I think this is because we delay LEDGER_STATUSs and CATCHUP_REPLYs for the first ledger, which is Pool now, so we get Checkpoints before we actually start catching up Audit ledger. So, we catch-up all the txns in Audit (and hence Domain), and this is not what we want to test.
Please make sure that we delay catchup msgs for Audit or Domain ledgers, so that we catchup less.

Also please make sure that other tests from this folder are OK.